tox.ini

```diff
@@ -1,19 +1,14 @@
 [tox]
-envlist = py312-pins-pytest_all,build_docs
+envlist = py3-pytest_skipslow
```

Comment:
I want to make it so that when we type `tox`, we end up running the test suite that we'll probably run most often.